### DISCUSSION TITLE
Manual Setup

### DISCUSSION THREAD

**Discussion by tanjaamplo** [Amplo-GmbH/Platform-Documentation#6] [Ideas]
![image](https://user-images.githubusercontent.com/91560861/135419916-0ec68d40-8c55-4514-89be-bb626d6f45d8.png)
This needs to sound more fun! My suggestion:
If you don’t have a structured data storage or ticket system, or simply want to try out the power of our AutoML, you can train and test our Automated Diagnostics service manually. We'll create together with one of your service engineers a list of occurrences and the corresponding measurement data snapshots for every incident.

**Comment by nielsuit227** [comment]
Haha fair enough. What about 
```
Even without structured data storage or ticket system you can do smarter maintenance!
Collect a list of incidents & occurrences with the corresponding measurement data. 
Upload it manually in the Data Overview and see the power of our AutoML! 
```

**Reply by tanjaamplo** [reply]
Perfect!
